template/gitlab-runner.tpl

```diff
@@ -20,22 +20,29 @@ EOF
 
 sed -i.bak s/__PARENT_TAG__/$PARENT_TAG/g /etc/gitlab-runner/config.toml
 
+${pre_install_certificates}
+
 # fetch Runner token from SSM and validate it
 token=$(aws ssm get-parameters --names "${secure_parameter_store_runner_token_key}" --with-decryption --region "${secure_parameter_store_region}" | jq -r ".Parameters | .[0] | .Value")
 
 valid_token=true
 if [[ "$token" != "null" ]]
 then
+<<<<<<< HEAD
```

Comment:
The PR contains several incorrect merges, please can you fix them.

Reply:
Ups, I guess it was me. Have to activate the `pre-commit` immediately ...

Reply:
One incorrect merge found and fixed.